ios/brave-ios/Tests/ClientTests/URLFormatTests.swift

```diff
@@ -12,22 +12,13 @@ import XCTest
 
 @MainActor class URLFormatTests: XCTestCase {
 
-  // Only init once
-  private static let icuInitialized = BraveCoreMain.initializeICUForTesting()
-
-  override func setUp() {
-    super.setUp()
-    assert(Self.icuInitialized, "ICU should load for test")
-  }
-
   func testURIFixup() {
     // Check valid URLs. We can load these after some fixup.
-    checkValidURL("about:", afterFixup: "about:")
-    checkValidURL("about:config", afterFixup: "about:config")
-    checkValidURL("about: config", afterFixup: "about:%20config")
+    checkValidURL("about:", afterFixup: "chrome://version/")
+    checkValidURL("about:config", afterFixup: "chrome://config/")
+    checkValidURL("about: config", afterFixup: "chrome://%20config/")
 
-    checkValidURL("file:///f/o/o", afterFixup: "file:///f/o/o")
```

Comment:
Hmm I dont see the replacement for this test? Any reason why we removed it ?

Reply:
Can't access file URLs on iOS via WebKit without a special request.
Also this automatically hits the search engine.